CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
```

Comment:
changelog is automatically generated
and since atm we don't have much new things in this repo I'd likely create a new release when a bigger feature gets merged anyway